### ISSUE TITLE
About the learning rate at ssl stage

### ISSUE THREAD

**Issue by Whale-ice** [Divadi/DetMatch#2]
Hello：
I am interested in ssl object detection and your work! 
I notice that you use adamW with onecycle at pretraining stage, but use the starting or max learning rate of pre-training as the constant learning rate when training 2D or 3D SSL. Why you don't use Adam with onecycle  at 3D ssl stage?
Thanks!
![image](https://user-images.githubusercontent.com/37449696/223927657-00540ac8-6c30-42c5-a7e1-b7614d8dc41f.png)
![image](https://user-images.githubusercontent.com/37449696/223927802-03bed768-fd29-4a53-9beb-9a1c0f951940.png)

**Comment by Divadi** [comment]
I followed Unbiased teacher's method of constant LR, then taking the EMA teacher as the final model. I found this to yield similar, but slightly more stable results early on.
